openedx/core/djangoapps/credit/tests/test_views.py

```diff
@@ -365,40 +367,18 @@ def _serialize_credit_course(self, credit_course):
             'enabled': credit_course.enabled
         }
 
-    def test_session_auth(self):
```

Comment:
@jimabramson updated. I think this kind of test amounts to testing DRF, but I've added it regardless.